server/camphoric/views.py

```diff
@@ -324,9 +322,12 @@ def post_registration(self, request, event):
         for camper in campers:
             camper.save()
 
+        # for some reason this isn't happening on its own
+        registration.refresh_from_db()
```

Comment:
Probably not necessary with the above approach, since the registration will have been saved with the correct pricing results the first time